### PR TITLE
test

### PR DESCRIPTION
**Pull request by homerours**

### CODE REVIEW

**Comment by filfat** [comment]
The plan were to have our own fork of this plugin for inhouse development hence why we even changed the name. We also added Windows support; if you could at least have the decency to not use the name we gave it would have been nice.

**Comment by homerours** [comment]
Thank you for finally explaining what you were doing, instead of randomly opening/closing issues/PR !

Thank you again for windows support. I kept your the "MusicController" name to maintain compatibility between my repo and your fork, so that future PR can happen.
Now that you created a new repo (without keeping any links with mine), it is not possible anymore (compatibility would have speed up developpment and improvements !! No need to do twice the same job...)... If you want me to free the "MusicController" name and switch back to "MusicControls", I can do it, no problem.

But I think you misunderstood the purpose of Github: "build software better, together". It is meant to be **collaborative software**. If you prefer "inhouse development", you better create a private repo (bitbucket do this for free!).

In conclusion, if you want that people have "at least the decency" to understand you acts, please have "at least the decency" to explain them.

**Comment by filfat** [comment]
Hi, didn't mean to come of assholish if I did :)

What I meant with "inhouse development" were that we would work on a a different version of your plugin (more focused on bringing certain features we need), it would still be public of course.

I started by opening two PRs with some fixes but later deleted the fork which meant I had to close the PRs. I opened the issues before I decided to create a new version of this plugin.


Sorry if I miss-understood why you merged the fork and good luck on further development.
Cheers.
